### DISCUSSION TITLE
Timezone email alerts (ElastAlert)

### DISCUSSION THREAD

**Discussion by DashCarlos** [Security-Onion-Solutions/securityonion#2172] [Unsupported Versions] [answered]
I have a little problem about the time zone in which the alerts (suricata) are registered. I have been able to change the time of the host but not the docker that has the alerts (I assume so-suricata or so-elastalert), so the alerts are one hour out of date.
Btw, the command I used on the host is _timedatectl set-timezone Europe/Madrid_, this command is not recognized in the so-elastalert docker

I have been able to change it on the docker so-suricata with the tzselect command and it has been changed, but the time is still out of date in the emails, I am going to suppose that I should change the time in so-elastalert as well, but I have looked for a lot of information and different ways of do it and I have not succeeded
Any ideas?

**Comment by dougburks** [comment, marked as answer]
Security Onion intentionally sets the OS timezone to UTC and we recommend you keep it that way:
https://docs.securityonion.net/en/2.3/timezones.html
